WORKFLOW-Heroku.md

```diff
@@ -0,0 +1,70 @@
+# Heroku Deployment for Rails
+####Pre-requisites:
+- Existing Rails App
+- Git & associated SSH Keys
+```
+# If app isn't associated with git yet
+$ git init
+$ git add .
+$ git commit -m 'Sets initial commit'
+```
+```
+# If no SSH Keys generated yet, run command below;
+# Fill-in succeeding prompts as desired but leave out passphrase prompt as empty
+$ ssh-keygen -t rsa
+```
+- Heroku Account. Sign up [here].
+- Heroku Toolbelt
+```
+# OSX (via Homebrew)
+$  brew install heroku-toolbelt
+# Debian/Ubuntu
+$ wget -qO- https://toolbelt.heroku.com/install-ubuntu.sh | sh
+# Login
+$ heroku login
+```
+
+
+####Setup:
+Specify ruby version in your `Gemfile`. This is clearly ok unless you indicate a different version from that of your `.ruby-version` et al
+```
+source "https://rubygems.org"
+ruby "2.1.2"
```

Comment:
ah of course, but adding whatever ORM gem in `Gemfile` is a mandatory part of starting a rails app and should already be there when one tackles deployment.